default.htm

```diff
@@ -1,255 +1,527 @@
 <!DOCTYPE html>
 <html lang="en" data-bs-theme="dark">
 
-	<head>
-		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<title>KTNE Solver</title>
-		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
-			integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-		<style>
-			* {
-				font-family: Consolas, 'Courier New', Courier, monospace;
-			}
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>KTNE Solver</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+		  integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+	<style>
+		* {
+			font-family: Consolas, 'Courier New', Courier, monospace;
+		}
+
+		body {
+			background-color: black;
+		}
+
+		.module {
+			min-width: 350px;
+		}
+
+
+		input[type='text'],
+		#solution {
+			color: rgb(0, 255, 0) !important;
+			text-transform: uppercase;
+		}
 
-			body {
-				background-color: black;
+			input[type='text']:focus {
+				border-color: #00FF00;
+				box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(0, 255, 0, 0.6);
 			}
 
-			.module {
-				min-width: 350px;
+		#button_colour1:not(:checked)+.btn {
+			color: #0d6efd;
+		}
+
+		#button_colour1:checked+.btn {
+			background-color: #0d6efd;
+		}
+
+		#button_colour2:not(:checked)+.btn {
+			color: #f8f9fa;
+		}
+
+		#button_colour2:checked+.btn {
+			background-color: #adb5bd;
+		}
+
+		#button_colour3:not(:checked)+.btn {
+			color: #ffc107;
+		}
+
+		#button_colour3:checked+.btn {
+			background-color: #ffc107;
+		}
+
+		#button_colour4:not(:checked)+.btn {
+			color: #dc3545;
+		}
+
+		#button_colour4:checked+.btn {
+			background-color: #dc3545;
+		}
+
+		.btn-group-sm {
+			margin-top: 6px; /*TODO : better solution to spacing*/
```

Comment:
There's a bootstrap way of doing this.

Reply:
Figured, I just forgot to check my TODOs